### PR TITLE
CEDS-1418 Requests Distinction

### PR DESCRIPTION
**Pull request by MaciejRewera**
This PR is introduced to allow differentiating between different submission types being made by the user.
To fulfill that, every request has its own endpoint on the backend, since this is the simplest solution. It involves almost no logic and is more clean than in-code-logic solution.

### CODE REVIEW

File: app/views/movements.scala.html [hunk 1]
```diff
@@ -36,18 +36,16 @@
...
-                <th class="table-cell" id="status">@messages("submissions.status")</th>
                 <th class="table-cell" id="noOfNotifications">@messages("submissions.noOfNotifications")</th>
             </tr>
 
             @for( (submission, notifications) <- submissions ){
                 <tr class="table-row">
-                    <td class="table-cell" id="ucr-@submission.conversationId">@submission.ucr</td>
-                    <td class="table-cell" id="submissionType-@submission.conversationId">@submission.submissionType</td>
-                    <td class="table-cell" id="submissionAction-@submission.conversationId">@submission.submissionAction</td>
+                    <td class="table-cell" id="ucr-@submission.conversationId">@submission.ucrBlocks.headOption.map(_.ucr).getOrElse("-")</td>
```
Comment (bambuchaAdm) [inline app/views/movements.scala.html]: And what with other ucr's blocks ?

Comment (MaciejRewera) [reply app/views/movements.scala.html]: Thanks for spotting this. Don't know if that will ever be the case, but added fix for that.

File: app/connectors/CustomsDeclareExportsMovementsConnector.scala [hunk 2]
```diff
@@ -33,47 +32,66 @@ class CustomsDeclareExportsMovementsConnector @Inject()(appConfig: AppConfig, ht
 
   private val logger = Logger(this.getClass)
 
-  private val MovementSubmissionUrl = s"${appConfig.customsDeclareExportsMovements}${appConfig.saveMovementSubmission}"
-  private val MovementConsolidationUrl =
-    s"${appConfig.customsDeclareExportsMovements}${appConfig.submitMovementConsolidation}"
+  private val CustomsDeclareExportsMovementsUrl = s"${appConfig.customsDeclareExportsMovements}"
+  private val ArrivalSubmissionUrl =
```
Comment (bambuchaAdm) [inline app/connectors/CustomsDeclareExportsMovementsConnector.scala]: why `val`s from capital letter ?

Comment (MaciejRewera) [reply app/connectors/CustomsDeclareExportsMovementsConnector.scala]: To indicate they're constants holding specific value.

**Pull request merged**